source/_posts/2022-12-20-year-of-voice.markdown

```diff
@@ -0,0 +1,65 @@
+
+If you follow the news, it might sound like voice assistants have failed. Amazon is set to [lose $10 billion on Alexa this year](https://arstechnica.com/gadgets/2022/11/amazon-alexa-is-a-colossal-failure-on-pace-to-lose-10-billion-this-year/) and is planning layoffs. Google too, is reducing its [support for Google Assistant](https://arstechnica.com/gadgets/2022/10/report-google-doubles-down-on-pixel-hardware-cuts-google-assistant-support/) as it’s trying to cut costs. The truth is that voice as the next computing platform that drives billions of dollars of extra revenue has failed. Instead, users mainly use their voice assistants to manage shopping lists, set timers, play music and control their home. Voice has failed being a source of revenue, it has not failed its users.
+
+With Home Assistant we’ve always been interested in voice. We used to work with [Snips](https://snips.ai/) back in the day, but they got acquired and shut down. We worked with Stanford on their [Almond/Genie platform](https://www.home-assistant.io/blog/2021/12/21/stanford-genie/), but it is a research driven project that never got production ready. And yes, you can use Home Assistant to send all your data to the clouds of Google and Amazon to leverage their voice assistants, but you shouldn't have to give up your privacy to just turn on the lights by voice.
+
+The most promising project out there is [Rhasspy](https://rhasspy.readthedocs.io/en/latest/) created by Mike Hansen. A project that allowed people to build their own voice assistant which could also tie into Home Assistant. Rhasspy stands out from other open source voice projects because Mike didn’t focus on just English. Instead, his goal has been to make it work for everyone and he already supports 16 different languages today.
+
+With Home Assistant we want to make a privacy and locally focused smart home available to everyone. Mike’s approach with Rhasspy aligns with Home Assistant, and so we’re happy to announce that Mike has joined Nabu Casa to work full-time on voice in Home Assistant.
+
+## Iterating in the open
+
+With Home Assistant we prefer to get the things we’re building in the user's hands as early as possible. Even basic functionality allows users to find things that work and don’t work, allowing us to address the direction if needed.
+
+A voice assistant has a lot of different parts: hot word detection, speech to text, intent recognition, intent execution, text to speech. Making each work in every language is a lot of work. The most important part is the intent recognition and intent execution. We need to be able to understand your commands and execute them.
+
+We started gathering these command sentences in our new [intents repository](https://github.com/home-assistant/intents). It will soon power the existing [conversation integration](https://www.home-assistant.io/integrations/conversation) in Home Assistant, allowing you to use our app to write and say commands.
```

Comment:
```suggestion
We started gathering these command sentences in our new [intents repository](https://github.com/home-assistant/intents). It will soon power the existing [conversation integration](/integrations/conversation) in Home Assistant, allowing you to use our app to write and say commands.
```